src/util/child.js

```diff
@@ -116,7 +121,19 @@ export function spawn(
           }
 
           if (proc.stdout) {
-            proc.stdout.on('data', updateStdout);
+            // Prefix output with workspace name, if we have it
+            if (opts.workspaceName) {
+              // Stay consistent by using the key that is mapped to proc
+              if (!workspaceColors[key]) {
+                const randInt = Math.floor(Math.random() * colors.length);
+                workspaceColors[key] = colors[randInt];
```

Comment:
Is there a reason not to just go through the colors array in order here?
It seems like by using `rand` we could potentially pick the same color for neighboring workspaces.

Reply:
I agree

Reply:
by going through the array it will also make the output deterministic